Replace branded button instead of appending it to avoid duplication

luizreis opened the pull request:
If the payment button is enabled on checkout, the branded button is duplicated every time checkout is re-calculated. This occurs because the button is being appended to the wrapper, instead of replacing its contents. This commit replaces the wrapper contents instead of appending the button, so it's not duplicated.

Fixes #1220.

#### Changes proposed in this Pull Request:

- Replace branded button in wrapper instead of appending it

#### Testing instructions:

1. Add the following filter to add the Payment Request Button to the checkout page:
`add_filter( 'wc_stripe_show_payment_request_on_checkout', '__return_true' );`
2. Set the option 'Payment Request Button Type' to 'Branded' in plugin settings
3. Add an item to the cart and go to the Checkout page
4. The Google Pay button should appear once
5. Toggle a checkout recalculation by changing the billing/shipping country
6. The Google Pay button should appear once

nabeel-s commented:
Although I'm not an official reviewer, I can confirm that I have applied the changes in the commit to my site and it works fine on my end.

nabeel-s commented:
Although I'm not an official reviewer, I can confirm that I have applied the changes in this commit to my site and it works fine on my end.

nabeel-s commented:
I can confirm that I have applied the changes in this commit to my site and it works fine on my end:

![checkout-button](https://user-images.githubusercontent.com/7054755/84746614-28173000-afae-11ea-9606-d4a36e4e009c.png)

v18 commented on assets/js/stripe-payment-request.js:
```diff
@@ -640,7 +640,7 @@ jQuery( function( $ ) {
 			} else if ( wc_stripe_payment_request.isBrandedPaymentRequestButton( prButton ) ) {
 				$( '#wc-stripe-payment-request-wrapper, #wc-stripe-payment-request-button-separator' ).show();
-				$( '#wc-stripe-payment-request-button' ).append( prButton );
+				$( '#wc-stripe-payment-request-button' ).html( prButton );
```
Looks good. I have just one question - from what I can see there can only ever be one button displayed. Is that assumption correct? If so, I think this is good to go.

luizreis replied:
Thanks for the review @v18! Yes, that's right. If there are multiple payment options, they can be selected in the Payment Request API modal once the button is clicked.

v18 commented:
@luizreis the change looks good & tests. I added just one question.

luizreis commented:
> @luizreis the change looks good & tests. I added just one question.

Awesome! I've just pushed an update to address the minified file changes and updated the `.pot` file, which had some changes from another PR.

v18 commented:
🚢
